### PR TITLE
Removed the custom `useragent` as it messes up some sites, including Google.

### PR DESCRIPTION
**Pull request by MystPi**
User experience is more important than a custom `useragent`, at least in my opinion.

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
             <input placeholder="Type URL" id="searchbox" name="searchbox" type="url">
             <button id="load">🔎</button>
         </nav>
-        <webview id="view" src="home.html" useragent="Freecat/1.1 Chromium/"></webview>
```
Comment (hello-smile6) [inline app/index.html:13]: Looks good, removing the UA string.

File: app/index.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
...
+        <webview id="view" src="home.html"></webview>
```
Comment (hello-smile6) [inline app/index.html:13]: No added changes. Nice! Sometimes people might add other things.

File: app/index.html [hunk 3]
```diff
@@ -10,7 +10,7 @@
...
         <script src="js/dynamic.js"></script>
     </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (hello-smile6) [inline app/index.html:16]: Is this what broke the current release?

**Review by hello-smile6 — APPROVED** [review]
Nice work, @MystPi

**Comment by JaydenDev** [comment]
merged

**Pull request merged**